sdk/storage/azure-storage-file-share/HISTORY.md

```diff
@@ -2,6 +2,15 @@
 
 ## Version 12.0.0b5:
 
+**Important: This package has been renamed**
```

Comment:
Technically, people that will read this readme will read the one of "azure-storage-file-share". So I would rephrase to something like "This package is the renaming of azure-storage-file" or something like that.